content/documentation.rst

```diff
@@ -62,20 +62,71 @@ semicolons, rather than after the Nth column.
     long lines.
 
 
-Writing User's Guides
-=====================
+About the Documentation Sections
+================================
+
+**Introduction**
```

Comment:
The 'Introduction' section is about this 'Introduction' page in the Documentation: https://www.sftpplus.com/documentation/sftpplus/latest/introduction.html